src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -0,0 +1,204 @@
+
+![Topology example](./images/topology-example.png "topology example")
+
+### Create vertices [#create-vertices]
+
+The following mutation creates one or more vertices that represent the entity and the source from which your incident comes from. 
+
+The NerdGraph call below uses the following fields:
+* `accountId`: Your New Relic account ID.
+* `name`: The name of the vertex. This value is case sensitive and must be unique within the graph.
+* `vertexClass`: The classification of this vertex. The class can be `application`, `host`, `cloud service`, `cluster`, or `datastore`. This classification allows your decision logic to restrict your topology-based correlation to restrict vertices matching these classifications.
+* `definingAttributes`: A unique set of attributes (key/value pairs) that match those from incident events. If an incident contains any of the key/value pairs of a vertex’s `definingAttributes`, it’s matched to that vertex. To learn more about attributes and how they’re added, see [Add attributes](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#add-attributes).
+
+Example call: 
+
+```
+mutation {
+  aiTopologyCollectorCreateVertices(
+  accountId: <var>NEW_RELIC_ACCOUNT_ID</var>, 
+    vertices: [
+    	{name: "ServiceA", vertexClass: APPLICATION, definingAttributes: [{key: "application/name", value: "ServiceA"}]}, 
+    	{name: "ServiceB", vertexClass: APPLICATION, definingAttributes: [{key: "application/name", value: "ServiceB"}]}, 
+    	{name: "ServiceC", vertexClass: APPLICATION, definingAttributes: [{key: "application/name", value: "ServiceC"}]}, 
+    	{name: "HOST1", vertexClass: HOST, definingAttributes: [{key: "host/name", value: "HOST1"}, {key: "availability-zone", value: "us-west-2a"}, {key: "region", value: "us-west-2"}]}, 
+    	{name: "HOST2", vertexClass: HOST, definingAttributes: [{key: "host/name", value: "HOST2"}, {key: "availability-zone", value: "us-west-2b"}, {key: "region", value: "us-west-2"}]}, 
+    	{name: "HOST3", vertexClass: HOST, definingAttributes: [{key: "host/name", value: "HOST3"}, {key: "availability-zone", value: "us-west-2c"}, {key: "region", value: "us-west-2"}]}]) 
+  {
+    result
+  }
+}
+```
+
+### Create edges [#edges]
+
+This mutation is used to create one or more edges that represent relationships between one vertex to another vertex.
+
+The NerdGraph call below makes use of these fields: 
+
+* `accountId`: Your New Relic account ID.
+* `fromVertexName`:  The name of the vertex you’re defining the edge for.
+* `toVertexName`: The name of the vertex at which you’re ending the edge.
+* `directed`: Optional. A boolean flag that describes how you want to connect one vertex to another vertex. True means the edge can traverse only one direction, indicating a one-way relationship. False means the edge can traverse in both directions, indicating a two-way relationship. By default, directed = `true`. 
```

Comment:
Is the edge the thing that traverses or the thing being traversed? Feels more like the latter to me, but then again, I may be conceptualizing all of this really wrongly.

How about simplifying this "True means the edge can traverse only one direction, indicating a one-way relationship. False means the edge can traverse in both directions, indicating a two-way relationship." to "Use True for one-way relationships and False for two-way relationships."